mne_bids_pipeline/tests/datasets.py

```diff
@@ -85,6 +85,8 @@ class DATASET_OPTIONS_T(TypedDict, total=False):
     "ds003775": {
         "openneuro": "ds003775",
         "include": ["sub-010"],
+        # See https://github.com/OpenNeuroOrg/openneuro/issues/2976
+        "exclude": ["sub-010/ses-t1/sub-010_ses-t1_scans.tsv"],
```

Comment:
No idea how long it'll take them to fix the issue so I'm going to see if we can just vendor the file in the meantime

Reply:
It seems to have worked!